providers/zendesksupport/read.go

```diff
@@ -26,7 +26,7 @@ func (c *Connector) Read(ctx context.Context, config common.ReadParams) (*common
 		return nil, err
 	}
 
-	responseFieldName := ObjectNameToResponseField.Get(config.ObjectName)
+	responseFieldName := ObjectNameToResponseField[c.Module.ID].Get(config.ObjectName)
```

Comment:
This could panic, can you add a check here?

Reply:
During development I thought so too and validated it. In golang it cannot panic.
The reason for this is every object I use is not a reference.

If `ObjectNameToResponseField` map hits a miss then the default empty value is returned, which will be empty `DefaultMap[string,string]` struct. Calling `Get` will look through empty map, then try to use `nil` fallback (there is a check) and would return the empty value of string which is `""`.

Worse case, it cannot break, panic.